acceptance/acceptance_test.go

```diff
@@ -368,8 +368,25 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 	err = CopyDir(dir, tmpDir, inputs, outputs)
 	require.NoError(t, err)
 
+	timeout := config.Timeout
+
+	if timeout == 0 {
+		if isRunningOnCloud {
+			timeout = 150 * time.Second
```

Comment:
Do we have any stats on how much time the slowest test takes right now in CI/CD? I find the timeout to be quite useful when authoring new tests that might be hanging, but at 150s it's almost the same as having no timeout.

Reply:
For integration tests, per each run, we have "Analyze slow tests" step in github workflow that shows slowest tests there. It's not aggregated further as far as I know.

Reply:
I've added similar stats for unit tests: https://github.com/databricks/cli/pull/2798